core-primitives/types/src/lib.rs

```diff
@@ -41,6 +42,7 @@ pub type MrEnclave = [u8; 32];
 pub type ConfirmCallFn = ([u8; 2], ShardIdentifier, H256, Vec<u8>);
 pub type ShieldFundsFn = ([u8; 2], Vec<u8>, Amount, ShardIdentifier);
 pub type CallWorkerFn = ([u8; 2], Request);
+pub type AckGameFn = ([u8; 2], GameEngine, Vec<H256>);
```

Comment:
I think this Fn is not correct any more, with the new pallet.. But you'll only need it in the next PR I suppose.

It now also expects a ShardIdentifier as fourth argument https://github.com/ajuna-network/ajuna-node/blob/update-substrate-5/pallets/ajuna-gameregistry/src/lib.rs#L316